static/css/ux-library.css

```diff
@@ -0,0 +1,584 @@
+}
+
+.button-med-alt{
+    width: 108px;
+    height: 51px;
+    flex-shrink: 0;
+    border: none;
+}
+
+.button-single-action{
+    width: 48px;
+    height: 48px;
+    flex-shrink: 0;
+    border-radius: 50px;
+    border: none;
+}
+
+.button-single-action-sm{
+    width: 32px;
+    height: 32px;
+    flex-shrink: 0;
+    border-radius: 50px;
+    border: none;
+}
+
+.button-single-action-chevron{
+    width: 36px;
+    height: 36px;
+    flex-shrink: 0;
+    border-radius: 50px;
+    border: none;
+}
+
+.button-single-action-arrow{
+    width: 36px;
+    height: 36px;
+    flex-shrink: 0;
+    border: none;
+}
+
+/*Body Components*/
+/*Custom Settings For Each Section*/
+.content-section{
+    margin-bottom: 120px;
+}
+
+.card-button-margin{
+    margin-top: 30px;
+}
+
+.technology-title{
+    min-width: 409px;
+}
+
+.technology-card{
+    margin-left: 10px;
+    margin-right: 10px;
+}
+
+.faq-card-content{
+    padding-left: 25px;
+    padding-right: 25px;
+    padding-top: 48px;
+    padding-bottom: 48px;
+}
+
+.usage-content-title{
+    margin-bottom: 20px;
+}
+
+.usage-content-button{
+    margin-top: 20px;
+}
+
+/*Get Started Users*/
+
+.usage-menu-container{
+    border-bottom: #000000;
+    border-bottom-style: solid;
+    border-bottom-width: thin;
+    margin-top: 80px;
+    margin-bottom: 50px;
+}
+.usage-menu-button{
+    font-family: 'Helvetica Neue', Helvetica, Arial,  sans-serif;
+    font-weight: 300;
+    font-size: 48px;
+    color: #000000;
+    text-decoration: none;
+    cursor: pointer;
+
+}
+
+.usage-menu-margin{
+    margin-right: 60px;
+    margin-left: 60px;
+}
+
+.getStartedOpacity{
+    background-color: rgba(0,0,0,0.5);
+}
+
+.student-user-content{
+    color: #FFFFFF;
+}
+
+.student-user-image{
+    width: 100%;
+    height: 842px;
+    background-image: url("../images/student-user.jpg");
+    background-position: center;
+    background-size: cover;
+}
+
+.researcher-user-content{
+    color: #FFFFFF;
+}
+
+.researcher-user-image{
+    width: 100%;
+    height: 842px;
+    background-image: url("../images/researcher-user.jpg");
+    background-position: center;
+    background-size: cover;
+}
+
+.hobbyist-user-content{
+    color: #FFFFFF;
+}
+
+.hobbyist-user-image{
+    width: 100%;
+    height: 842px;
+    background-image: url("../images/hobbyist-user.jpg");
+    background-position: center;
+    background-size: cover;
+}
+
+/*Extras*/
+
+/*Feedback*/
+
+.feedback-card{
+    margin-bottom: 50px;
+}
+
+/*Useful Link*/
+
+.useful-link{
+    margin-bottom: 30px;
+}
+
+/*Social Media*/
+
+.social-media-section{
+    margin-top: 120px;
+}
+.social-media-icon{
+    height: 40px;
+    width: 40px;
+    margin-right: 30px;
+}
+
+/*Footer*/
+
+.footer{
+    background-image: url("../images/footer/world-relief-map.jpg");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding-top: 100px;
+    padding-bottom: 100px;
+}
+
+.funder-logo{
+    min-width: 10%;
```

Comment:
min-width should be a fixed value. 10% of a tiny screen is potentially unreadable

Reply:
Changed min width to 100px.